spec/unit/queue_storage_spec.rb

```diff
@@ -22,7 +22,7 @@ module Backend
 
           context 'with a invalid configuration' do
             it 'returns an exception' do
-              expect { conn }.to raise_error RuntimeError
+              expect { conn }.to raise_error
```

Comment:
This has been changed because the RuntimeError is not raised anymore after the changes that have been done in the program.

We know that it should crash but without specifying any specific error now